tests/geometry/functional/test_so3.py

```diff
@@ -19,7 +19,20 @@ def test_exp(batch_size: int, dtype: torch.dtype):
     tangent_vector = torch.rand(batch_size, 3, dtype=dtype, generator=rng)
 
     # check analytic backward for the operator
-    check_lie_group_function(so3, "exp", TEST_EPS, tangent_vector)
+    check_lie_group_function(so3, "exp", 1e-6, (tangent_vector,))
```

Comment:
Hardcoded `1e-6`.

Reply:
Probably fine? I think we have a bunch of tolerances hardcoded in many unit tests.